static/js/lib/validation.js

```diff
@@ -91,12 +97,32 @@ export const postURLValidation = validateIfPostType(
   }
 )
 
+const isEmptyArticle = (article: Array<Object>): boolean => {
+  // unfortunately detecting an empty article is non-trivial :/
+
+  // if it's an empty array that's easy
+  if (R.equals(article, [])) {
+    return true
+  }
+
+  // sometimes the editor will leave one empty tag in it
+  if (article.length === 1) {
+    const [obj] = article
+    if ((R.equals(obj.attributes, {}), R.equals(obj.children, []))) {
```

Comment:
Was the intent to use the comma operator here? It will ignore the left item and only return the result of the right item

Reply:
ah no that should have been `||`, which I'm pretty sure I wrote...wondering if the formatter had other ideas